website/docs/connect-data/how-to-guides/fetch-and-filter-data-in-sql.md

```diff
@@ -0,0 +1,255 @@
+* Do not wrap the mustache binding `{{}}` within quotes.
+
+
+</dd>
+
+<dd>
+
+**Prepared statements disabled**
+
+
+
+```sql
+-- With prepared statements disabled
+SELECT * FROM public."features" where name ILIKE '%{{search_input.text}}%' order by created_at desc
+```
+
+
+Placing the `%`character within single quotes can lead to unexpected results and security vulnerabilities. It doesn't provide the same level of security and flexibility as the recommended format, making it more susceptible to SQL injection attacks. 
+
+</dd>
+
+
+
+
+## Working with the IN 
+
+The IN clause in SQL allows you to filter results based on multiple values. To use it correctly with prepared statements, follow these steps:
+
+
+<dd>
+
+**Prepared statements enabled** (recommended)
+
+
+```sql
+-- With prepared statements enabled
+SELECT * FROM products WHERE product_id IN ({{productID1}}, {{productID2}}, ...);
+```
+
+* Each value in the list should be provided as a separate mustache binding `{{}}` value.
+* Do not wrap the mustache binding `{{}}` within quotes.
+
+</dd>
+
+<dd>
+
+**Prepared statements disabled**
+
+```sql
+-- With prepared statements disabled
+SELECT * FROM products WHERE product_id IN {{productIDList}};
+-- productIDList = ["productID1", "productID2", ...]
+```
+Placing multiple product IDs within a single mustache binding can lead to unexpected results and security vulnerabilities.
+
+</dd>
+
+## Alternative to IN with ANY
+
+When working with IN statements, you may encounter situations where it's more convenient to use the ANY statement. The ANY keyword provides an alternative approach to achieving the same result.
+
+* When using this approach, provide all individual values in the list together, without separating them.
+* Do not wrap the binding within quotes.
+
+<dd>
+
+**Prepared statements enabled** (recommended)
+
+
+```sql
+//Recommended format:
+SELECT * FROM public."employees" where name=ANY ({{myBinding.data}});
+
+//Post evaluation it should look like:
+SELECT * FROM public."employees" where name=ANY ({{'{name, dept}'}});
+```
+
+This query uses the ANY keyword to match the `name` column in the `employees` table with both the `name` and `dept` values together in a structured array.
+
+
+</dd>
+
+
+## Handling IS Keyword in Prepared Statements
+
+When working with prepared statements, the IS keyword is not supported. Instead, you should rewrite your queries using the `=` operator. 
+
+
+
+<dd>
+
+**Recommended**
+
+```sql
+-- With prepared statements disabled
+SELECT * FROM student WHERE pass = {{true}};
+```
+
+Using the `=` operator for comparisons ensures that your query works as expected with prepared statements turned on.
+
+
+</dd>
+
+<dd>
+
+**Avoid**
+
+```sql
+-- With prepared statements disabled
+SELECT * FROM student WHERE pass IS {{true}};
+```
+
+Using the IS keyword in your queries with prepared statements can lead to syntax errors. It's essential to switch to the = operator to maintain query integrity and functionality.
```

Comment:
`Using the IS keyword in your queries with prepared statements can lead to syntax errors. It's essential to switch to the = operator to maintain query integrity and functionality.` - As per my understanding this is not correct. The reason we don't support `IS` keyword is because of some bug in the MySQL database driver the we used to have.